packages/forms/resources/js/components/file-upload.js

```diff
@@ -802,6 +802,7 @@ import uk from 'filepond/locale/uk-ua'
 import vi from 'filepond/locale/vi-vi'
 import zh_CN from 'filepond/locale/zh-cn'
 import zh_TW from 'filepond/locale/zh-tw'
+import zh_HK from 'filepond/locale/zh-hk'
```

Comment:
Can you remove this in the meantime before Filepond merges please, otherwise we can't compile this file